sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionData.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Buffers.Text;
+using System.Runtime.Serialization;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// Represents the encryption data that is stored on the service.
+    /// </summary>
+    [DataContract]
+    public struct EncryptionData
+    {
+
+        /// <summary>
+        /// The blob encryption mode.
+        /// </summary>
+        [DataMember]
+        public string EncryptionMode { get; set; }
```

Comment:
If the type has to be public, do the setters at least?

Reply:
Should this be an enum?

Reply:
My understanding is that serialization packages need access to setters to deserialize into these objects. Unless I write my own custom deserializer for the whole thing, in which case why am I even using the deserialization package?